running.md

```diff
@@ -247,3 +247,24 @@ sudo systemctl enable --now cockpit.socket
 ```
 
 If the first command fails with "database file for ... does not exist", refresh/update your system with `sudo pacman -Syu` first.
+
+
+### openSUSE Tumbleweed
+{:#tumbleweed}
+
+[Cockpit](https://software.opensuse.org/package/cockpit) is included in [openSUSE Tumbleweed](https://www.opensuse.org/):
```

Comment:
Perhaps link to https://software.opensuse.org/package/cockpit ? Then viewers can directly see current versions and such.

Reply:
That's where it already links to :) Changed the second link to point to TW though